web/client/selectors/localizedLayerStyles.js

```diff
@@ -26,6 +26,17 @@ const {currentLocaleLanguageSelector} = require('./locale');
  */
 const isLocalizedLayerStylesEnabledSelector = (state) => has(state, 'localConfig.localizedLayerStyles');
 
+/**
+ * selects localizedLayerStyles value from state
+ * @memberof selectors.localizedLayerStyles
+ * @param  {object} state the state
+ * @return {boolean} boolean if the localizedLayerStyles property is defined
+ */
+const isLocalizedLayerStylesEnabledDashboardsSelector = (state) => {
+    return isArray(state?.localConfig?.plugins?.dashboard)
+    && state.localConfig.plugins.dashboard.find(item => item.name === 'DashboardEditor').cfg.catalog.localizedLayerStyles;
+};
+
```

Comment:
Since you have to backport it back where the ?. was not supported i would suggest to chagnes this selector to be backward compatible.

something like 
```suggestion
const isLocalizedLayerStylesEnabledDashboardsSelector = (state) => {
 const dashboardPlugins = get(state, "state.localConfig.plugins.dashboard"); // get of lodash
  const dashboardEditorPlugin = find(dashboardPlugins , item => item.name === 'DashboardEditor'); // find of lodash
    return get(dashboardEditorPlugin, "cfg.catalog.localizedLayerStyles", false); // get of lodash
};

```
adjust indentation and remove comments